docs/components/LessStyleGuideView.jsx

```diff
@@ -1,14 +1,233 @@
 import React from "react";
 
+import Example from "./Example";
 import View from "./View";
+import {Grid} from "src";
 
 
 export default function LessStyleGuideView() {
   const {cssClass} = LessStyleGuideView;
+  const {Col, Row} = Grid;
 
   return (
     <View className={cssClass.CONTAINER} title="Less Style Guide">
-      TODO
+      <Grid>
+        <Row grow className="margin--bottom--xl">
+          <Col span={6} className="self--start padding--right--l">
+            <h2>Naming Conventions</h2>
+            <p>Dewey uses a mix of BEM and Atomic CSS for naming conventions. It is influenced by&nbsp;
+            <a href="http://basscss.com/">Basscss</a> for utility classes, while components are
+            written with a BEM notation.</p>
+
+            <Example
+              title="Variables"
+              code={`
+                  @zIndex--1: 100;
+              `}
+            >
+              Variables are the most atomic building blocks of Dewey. Each variable defines a single value for a
+              specific attribute. Variables are lower-cased camelCase with a double dash <code>--</code> for modifiers.
+            </Example>
+
+            <Example
+              title="Mixins"
+              code={`
+                .text--bold {
+                   font-weight: bold;
+                   font-style: normal;
+                }
+              `}
+            >
+              Mixins are the next layer of Dewey, they assemble attributes into meaningfully grouped elements
+              that are readable and composable. Mixins are lower-cased camelCase with a double dash <code>--</code>
+              between modifiers.
+            </Example>
+
+            <Example
+              title="Mixin functions"
+              code={`
+                .border--x--s(@color) {
+                  .border--left--s(@color);
+                  .border--right--s(@color);
+                }
+              `}
+            >
+              We use mixin functions for more complex attribute collections that should be similarly reusable and
+              composable. Mixin functions are lower-cased camelCase with a double dash <code>--</code>
+              between modifiers.
+            </Example>
+
+            <Example
+              title="Components"
+              code={`
+                .TabBar--Tab--selected {
+                  .text--medium;
+                  .border--bottom--m(@primaryBlue);
+                  box-sizing: border-box;
+                  color: @primaryBlue;
+                  cursor: pointer;
+                }
+              `}
+            >
```

Comment:
Perhaps more complex examples should reference previous simpler examples (eg. `.text--bold` here?